config/crd/bases/app.k8s.io_applications.yaml

```diff
@@ -471,7 +473,8 @@ spec:
               format: int64
               type: integer
           type: object
-      type: object
+      required:
```

Comment:
I have to remove `type: object`, otherwise the test would fail with the error: `must only have "properties",  "required" or "description" at the root if the status subresource is enabled occurred`

Reply:
The above validation was from an older version of kubernetes: v1.11.0. I'm going to update k8s to 1.16.4 in the test. That should fix the validation failure without removing `type: object`.

Reply:
I think that sounds ok. In general we should make sure it runs on the last 3-4 versions of Kubernetes. Clusters with older versions than that are no longer supported by the community, so we can forget about them.